### PR TITLE
Feat: heatmap/basic

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

image

#### with label

image

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: src/store/heatmapAxes.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+  const tickIntervalCount = labels.length;
+  const tickDistance = tickIntervalCount ? axisSize / tickIntervalCount : axisSize;
+  const labelDistance = axisSize / labels.length;
+
+  return {
+    labels,
+    pointOnColumn: true,
+    isLabelAxis,
+    tickCount: labels.length + 1,
```
Comment (lja1018) [inline src/store/heatmapAxes.ts:23]: `labels.length`를 빼서 쓸 수 있을거 같아요!

**Review by lja1018 — APPROVED** [review]
[09/21] 리뷰완료
고생하셨습니다!

**Review by jungeun-cho — APPROVED** [review]
[09/21] 리뷰완료합니다.
히트맵 차트는 selectable 옵션을 지원안하는 이유가 있을까요?

File: types/options.d.ts [hunk 2]
```diff
@@ -1,8 +1,10 @@
-import { RawSeries } from '@t/store/store';
+import { Categories, RawSeries } from '@t/store/store';
 import { TooltipModel } from '@t/components/tooltip';
 export type RangeDataType<T> = [T, T];
 export type BoxSeriesDataType = number | RangeDataType<number>;
 type LineSeriesDataType = number | Point | [number, number] | [string, number];
+type HeatmapSeriesDataType = number[];
+export type HeatmapCategoriesType = { x: string[]; y: string[] };
```
[the remark this answers is not in the document]
Comment (jung-han) [reply types/options.d.ts:7]: 이건 옵션 브랜치에서 작업할게요!

[the remark this answers is not in the document]
Comment (jung-han) [reply types/options.d.ts:7]: selectable도 옵션 브랜치에서 작업하겠습니다~

**Review by shiren — APPROVED** [review]
수고하셨어요

**Pull request merged**